README.md

```diff
@@ -344,8 +344,35 @@ For a comparison of some of these, see this
 
 ## Storage Backends
 
-In order to not download or scan any previously scanned sources again, the _scanner_ can use a storage backend to store
-scan results for later reuse.
+In order to not download or scan any previously scanned sources again or to reuse scan results generated via other
+services, the _scanner_ can be configured to use so-called storage backends. Before processing a package, it checks
+whether compatible scan results are already available in one of the storages declared; if this is the case, they
+are downloaded and reused. Otherwise, the package is scanned now. Afterwards, the new results can be put into a
+storage for later reuse.
+
+It is possible to configure multiple storages to read scan results from and to write new results to independently.
+In this configuration, the order of the storages is relevant, too. This allows a fine-grained control over the
```

Comment:
I believe we should rewrite parts of this text to emphasize that the order only matters for *reading* (has the first match wins), but for *writing* **all** storages are used (and need to succeed).

Maybe: "For reading, the declaration order in the configuration is important as the first compatible match from a storage is used. This allows ..."

Then a paragraph after line 360 about the "write" case.

Reply:
Reworded and added a paragraph for the write case.